website/src/pages/Blog/styles.ts

```diff
@@ -131,47 +131,61 @@ const useStyles = makeStyles((theme: Theme) => ({
     fontWeight: 700,
   },
   linkBtn: {
-    textDecoration: 'none !important',
+    textDecoration: "none !important",
   },
   arrow: {
     marginLeft: theme.spacing(1),
   },
   tag: {
-     fontSize: 16,
-     width: 'fit-content',
-     padding: theme.spacing(.1, 4),
-     borderRadius: '8px',
-     lineHeight: '8px',
-     background: theme.palette.warning.light,
-     color: theme.palette.text.hint,
+    fontSize: 16,
+    width: "fit-content",
+    padding: theme.spacing(0.1, 4),
+    borderRadius: "8px",
+    lineHeight: "8px",
+    background: theme.palette.warning.light,
+    color: theme.palette.text.hint,
   },
   small: {
-    width: '32px',
-    height: '32px'
+    width: "32px",
+    height: "32px",
   },
   actionWrapper: {
-   justifyContent: 'space-between',
-   marginTop: 'auto'
-  },
-  author: {
-  display: 'flex',
-  alignItems: 'center',
+    justifyContent: "space-between",
+    marginTop: "auto",
+    [theme.breakpoints.down("sm")]: {
+      display: "block",
+    },
   },
   pagination: {
-    display: 'flex',
-    justifyContent: 'center',
+    display: "flex",
+    justifyContent: "center",
     paddingTop: theme.spacing(2),
-    '& .Mui-selected':{
+    "& .Mui-selected": {
       color: theme.palette.text.hint,
-      backgroundColor: 'transparent !important'
+      backgroundColor: "transparent !important",
+    },
+    "& .MuiPaginationItem-root": {
+      fontWeight: "bold",
     },
-    '& .MuiPaginationItem-root':{
-      fontWeight: 'bold'
-    }
   },
   cardContent: {
-    padding: theme.spacing(4, 0)
-  }
+    padding: theme.spacing(4, 0),
+  },
+  rightSpacing: {
+    border: "none",
+    background: "none",
+    marginRight: theme.spacing(1),
+    cursor: "pointer",
+  },
+  tabWapper: {
+    display: "flex",
+    overflow: "scroll",
+    width: "70%",
+    [theme.breakpoints.down("sm")]: {
+      display: "inlineBlock",
```

Comment:
```suggestion
      display: "inline-block",
```

Reply:
material UI supports camelCase, it won't support` -`

Reply:
That is for the left-hand side CSS property not for the value